test/e2e-ocl/helpers_test.go

```diff
@@ -696,36 +696,6 @@ func convertFilesFromContainerImageToBytesMap(t *testing.T, pullspec, containerF
 	return out
 }
 
-// Copy the entitlement certificates into the MCO namespace. If the secrets
-// cannot be found, calls t.Skip() to skip the test.
-//
-// Registers and returns a cleanup function to remove the certificate(s) after test completion.
-func copyEntitlementCerts(t *testing.T, cs *framework.ClientSet) func() {
-	src := metav1.ObjectMeta{
-		Name:      "etc-pki-entitlement",
-		Namespace: "openshift-config-managed",
-	}
-
-	dst := metav1.ObjectMeta{
-		Name:      src.Name,
-		Namespace: ctrlcommon.MCONamespace,
-	}
-
-	_, err := cs.CoreV1Interface.Secrets(src.Namespace).Get(context.TODO(), src.Name, metav1.GetOptions{})
-	if err == nil {
-		return cloneSecret(t, cs, src, dst)
-	}
-
-	if k8serrors.IsNotFound(err) {
-		t.Logf("Secret %q not found in %q, skipping test", src.Name, src.Namespace)
-		t.Skip()
```

Comment:
thought: I feel like we might still want to skip the test if we know it will fail due to a missing entitlement. I suppose now it would look for our clone in the MCO namespace as opposed to the original in the `openshift-config-managed` namespace. Instead, we can make the `useEtcPkiEntitlement` check be something like, `entitlementRequired`.